### PR TITLE
add command line and sentry button to show links, enable GitLab

### PR DESCRIPTION
**Pull request by vanesa**
"(Show/Hide) log links" action in command palette and a Sentry icon action item on the editor/title, update README and enable for GitLab.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
 [![codecov](https://codecov.io/gh/sourcegraph/sentry/branch/master/graph/badge.svg)](https://codecov.io/gh/sourcegraph/sourcegraph-sentry)
 
-# WIP: Sentry Sourcegraph extension
+# WIP: Sentry extension
```
Comment (lguychard) [inline README.md]: Should this still say WIP?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 
 Sentry helps devs track, organize and break down errors more efficiently, facilitating their debug process. We want to make it more convenient for developers to access Sentry's error tracking tools directly from the code that is doing the error handling, code such as `throw new Error(QUERY)`, `console.log(QUERY)`, `console.error(QUERY)` etc..
 
-In this first version, the Sentry extension will render a `View logs in Sentry` link on each line it detects such error handling code, leading the devs directly to the corresponding Sentry issues stream page.
+In this first version, the Sentry extension will render a `View logs in Sentry` link on each line it detects such error handling code, leading the devs directly to the corresponding Sentry issues stream page. Links will be rendereed when viewing files on [Sourcegraph](https://sourcegraph.com), GitHub and GitLab.
```
Comment (lguychard) [inline README.md]: Should this be in the future tense? also s/rendereed/rendered/

```suggestion
The Sentry extension renders a `View logs in Sentry` next to error throwing statements, linking directly to the corresponding Sentry issues stream page. Links are rendered when viewing files on [Sourcegraph](https://sourcegraph.com), GitHub and GitLab. 
```

File: src/extension.ts [hunk 3]
```diff
@@ -18,9 +18,7 @@ interface Params {
     file: string | null
 }
 
-const DECORATION_TYPE = sourcegraph.app.createDecorationType()
-const SETTINGSCONFIG = resolveSettings(sourcegraph.configuration.get<Settings>().value)
-const SENTRYORGANIZATION = SETTINGSCONFIG['sentry.organization']
+const SENTRYORGANIZATION = resolveSettings(sourcegraph.configuration.get<Settings>().value)['sentry.organization']
```
Comment (lguychard) [inline src/extension.ts]: Do you still need this?

File: src/extension.ts [hunk 4]
```diff
@@ -41,8 +39,14 @@ const COMMON_ERRORLOG_PATTERNS = [
     // java
     /logger\.(debug|error)\(['"`]([^'"`]+)['"`]\);/gi,
 ]
+const DECORATION_TYPE = sourcegraph.app.createDecorationType()
 
 export function activate(context: sourcegraph.ExtensionContext): void {
+    // TODO(lguychard) sourcegraph.configuration is currently not rxjs-compatible.
```
Comment (lguychard) [inline src/extension.ts]:
```suggestion
    // TODO: Change this when https://github.com/sourcegraph/sourcegraph/issues/3557 is resolved
```

File: src/extension.ts (hunk 3 above)
Comment (vanesa) [reply src/extension.ts]: Yes, for `decorateLine()` and `buildUrl`.

**Pull request merged**